web/package.json

```diff
@@ -10,12 +10,12 @@
         "build-locales:repair": "prettier --write ./src/locale-codes.ts",
         "esbuild:build": "node build.mjs",
         "esbuild:build-proxy": "node build.mjs --proxy",
-        "esbuild:watch": "node build.mjs --watch",
```

Comment:
we should probably either keep the node commands and add equivalents with bun and update the docs to say that bun is required for frontend (in that case we should probably switch everything to bun though)